mc/gui/main_win.py

```diff
@@ -109,13 +109,23 @@ def _setup_configure_active_breathing_phrase(self, panel_vbox_l4):
 
     def _setup_add_breathing_phrase_list_to_main_container(self, main_container_hbox_l3):
         breathing_phrase_list_vbox_l4 = self._setup_new_panel_in_main_window(main_container_hbox_l3)
+
+        title_qll = QtWidgets.QLabel("Breathing")
+        breathing_phrase_list_vbox_l4.addWidget(title_qll)
+        title_qll.setFont(mc.mc_global.get_font_large())
```

Comment:
I think it would be nice if the title would be bold, like where it says "Enabled". Also, it would be nice if it would be aligned with the box below.

Reply:
@mbed67 Bold added, and i've aligned (even though i had to do this "manually", not sure why, but i've written a note about it)